src/model/consent/consent.ts

```diff
@@ -108,6 +110,11 @@ export class ConsentDB {
         throw new NotFoundError('Consent', consent.id)
       }
 
+      // Cannot overwrite REVOKED status Consent
```

Comment:
This is business logic - shouldn't it be in the domain part of code. @jgeewax

Reply:
I agree - - however this is one of those cases where we previously decided that we don't want the model to be able to do something potentially dangerous (see 120/127 where we ensure we don't overwrite ACTIVE credentialStatus)